editor/editor_file_system.cpp

```diff
@@ -2398,18 +2398,22 @@ void EditorFileSystem::reimport_files(const Vector<String> &p_files) {
 	int from = 0;
 	for (int i = 0; i < reimport_files.size(); i++) {
 		if (groups_to_reimport.has(reimport_files[i].path)) {
+			from = i + 1;
 			continue;
 		}
 
 		if (use_multiple_threads && reimport_files[i].threaded) {
-			if (i + 1 == reimport_files.size() || reimport_files[i + 1].importer != reimport_files[from].importer) {
+			if (i + 1 == reimport_files.size() || reimport_files[i + 1].importer != reimport_files[from].importer || groups_to_reimport.has(reimport_files[i + 1].path)) {
 				if (from - i == 0) {
 					// Single file, do not use threads.
 					pr.step(reimport_files[i].path.get_file(), i);
 					_reimport_file(reimport_files[i].path);
 				} else {
 					Ref<ResourceImporter> importer = ResourceFormatImporter::get_singleton()->get_importer_by_name(reimport_files[from].importer);
-					ERR_CONTINUE(!importer.is_valid());
+					if (importer.is_null()) {
+						from = i + 1;
+						ERR_CONTINUE_MSG(true, vformat("Invalid importer for \"%s\".", reimport_files[from].importer));
```

Comment:
```suggestion
						ERR_CONTINUE_MSG(true, vformat("Invalid importer for \"%s\".", reimport_files[from].importer));
						from = i + 1;
```
I'm assuming, as the checked importer is the earlier `from`

Reply:
From my perspective, if the `from = i + 1;` is done after the ERR_CONTINUE_MSG, it will not be executed and next resource (which likely does not have the same importer) will be imported with the previous invalid importer. That was exactly the problem in https://github.com/godotengine/godot/pull/85130 .

Reply:
That's true, then you need to fetch the old value instead, my bad, but you need to handle that, because this will print the wrong message

Reply:
Oh!! That's totally right! I'll fix that!

Reply:
```suggestion
						ERR_PRINT(true, vformat("Invalid importer for \"%s\".", reimport_files[from].importer));
						from = i + 1;
						continue;
```

Reply:
Corrected as the original had a leftover `true`

Reply:
Done